### ISSUE TITLE
Support line_statement_prefix which is usually the suggested `#`

### ISSUE THREAD

**Issue by lipis** [wholroyd/vscode-jinja#9]
Can you add support for the line statements: http://jinja.pocoo.org/docs/dev/templates/#line-statements

```python
app.jinja_env.line_statement_prefix = '#'
app.jinja_env.line_comment_prefix = '##'
```

<img width="544" alt="screen shot 2016-11-05 at 21 05 25" src="https://cloud.githubusercontent.com/assets/125676/20033186/9f5695ca-a39b-11e6-9efc-54ac07ff0cf4.png">

**Comment by lipis** [comment]
Is this coming anytime soon?! :) 🏜 🐪 🌃

**Comment by wholroyd** [comment]
Yeah, sorry about the wait. I did the work for it which was included with other changes I was working on and then ran into issues mixing Jinja with other languages. Then gave up. I’ll have to go pull this changes out and push it. Least I can do for the time being. Give me a week to find some time to do the work.

**Comment by lipis** [comment]
No rush of course.. but I'll check in a week if there is a new version in my editor :)
